README.md

```diff
@@ -74,6 +74,11 @@ For 64-bit Linux users,  ```libc6:i386```, ```libstdc++6:i386```, ```libncurses5
 
 __NOTE__: To roll back to the original driver go to: Device Manager -> Right click on device -> Check box for "Delete the driver software for this device" and click Uninstall
 
+### Selecting a SoftDevice
+Softdevices contain the BLE stack and housekeeping, and must be downloaded once before a sketch can be loaded.
```

Comment:
They don't have to be downloaded if you don't plan to use the functionality from the softdevice. For normal Arduino usage you don't have to have a soft device present.